frontend/src/pages/dashboard/Dashboard.tsx

```diff
@@ -1,18 +1,15 @@
 import React from "react";
 import "./Dashboard.css";
-import { Link } from "react-router-dom";
+import { Header, MenuDesktop } from "../../IndexForImport";
 
 const Dashboard = () => {
   return (
-    <div>
-      <h2>This is Dashboard page</h2>
-      <div className="link-container">
-        <Link to="/project">Project</Link>
-        <Link to="/mytask">My Task</Link>
-        <Link to="/activity">Activity</Link>
-        <Link to="/team">Team</Link>
-        <Link to="/message">Message</Link>
-        <Link to="/setting">Setting</Link>
+    <div className="pages">
+      <div className="menu-desktop">
+        <MenuDesktop />
+      </div>
+      <div className="main">
+        <Header />
       </div>
```

Comment:
Every file from this and down is not correct, you are duplicating this logic in all the components. You should have used router outlet to render the internal part of the routing system (https://reactrouter.com/en/main/start/tutorial#nested-routes)